config.yml

```diff
@@ -658,6 +658,42 @@ nodes:
             bar(...)
             ^^^^^^^^
           end
+  - name: SplatArgumentsNode
+    child_nodes:
+      - name: operator
+        type: token
+      - name: expression
+        type: node
+    location: operator->expression
+    comment: |
+      Represents splat method arguments.
+
+          bar(*args)
+          ^^^^^^^^^^
+  - name: DoubleSplatArgumentsNode
```

Comment:
Looks like folks are calling this `kwsplat` (parser gem/ripper). Could we call this `KeywordStarNode` to keep that spirit here?